index.js

```diff
@@ -118,7 +193,8 @@ function myMap(array, callback) {
 // // => ["Hi, Raffy!", "Hi, Chase!"]
 // console.log("%c----------", "color: red") 
 
-
+function greet(name) {
+  return `Hi, ${name}!`
 
 // ***** Scope & Closures *****
 
```

Comment:
- For the first scope question, the key is to use a global variable. So try declaring `ticketNumber` outside of the function and updating it appropriately inside of the function 🤔

- The scope ones are tricky. Especially the bonus one! In order to help understand the the bonus one, check this Scrimba out https://scrimba.com/scrim/cLgEmZUZ and see if you can figure it out after watching it and playing around with the examples that Ian gives. (In Scrimba, you can pause the video and play around with the code shown on the screen. Just pause the video and click into the screen and start typing or running the code. Then, press play when you're ready to continue on.)